### ISSUE TITLE
Request for more detail for pre-training on ImageNet-21k

### ISSUE THREAD

**Issue by junsukchoe** [google-research/big_transfer#26]
Hi there!

I'd like to know the detail about how to train the network on ImageNet-21k. I carefully checked the paper and this repository, and I discovered lots of detail, but I cannot find which loss function is used for ImageNet-21k. In addition, I wonder how you process the multi-class labels for training. And also, it would be great if you give a detail of how to handle the unbalanced data distribution of ImageNet-21k. Could you tell me a bit more about those? 

Thanks,

**Comment by kracwarlock** [comment]
Also can you provide details on what data pre-processing was used for pre-training?

**Comment by lucasb-eyer** [comment]
Sure thing.

1. The loss function is sigmoid cross-entropy, i.e. a binary classifier for each label.
2. I'm not sure what you mean by "process the labels for training".
3. No special handling of imbalanced data.
4. Pre-processing for pre-training was standard "inception crop" resized to 224x224, random left/right flip, and rescale pixel values to [-1, 1]

hope this helps!

**Comment by junsukchoe** [comment]
Thanks a lot!

**Comment by kracwarlock** [comment]
@lucasb-eyer Can you clarify standard inception crop? For training and evaluation both?

**Comment by kracwarlock** [comment]
Also how did you create the train, valid and test splits?
